cwms-data-api/src/main/java/cwms/cda/data/dao/LocationGroupDao.java

```diff
@@ -352,18 +370,24 @@ public List<LocationGroup> getLocationGroups(String groupOfficeId,
             condition = condition.and(alcg.CAT_DB_OFFICE_ID.eq(categoryOfficeId.toUpperCase()));
         }
 
-        if (groupOfficeId != null) {
-            if (CWMS.equalsIgnoreCase(groupOfficeId)) {
+        if (officeId != null) {
```

Comment:
for consistent naming, use "locationOfficeId"

Reply:
renamed where appropriate